.github/workflows/tests.yaml

```diff
@@ -17,7 +17,8 @@ jobs:
       - name: Run tests
         run: make test
         env:
-          ETH_RPC_URL: ${{ secrets.ETH_RPC_URL }}
-          OPT_RPC_URL: ${{ secrets.OPT_RPC_URL }}
-          ARB_RPC_URL: ${{ secrets.ARB_RPC_URL }}
-          ARB_NOVA_RPC_URL: ${{ secrets.ARB_NOVA_RPC_URL }}
+          FOUNDRY_ROOT_CHAINID: 1
```

Comment:
This is good for now, but I think we should also rename the envs in github. We can do that later.